googleMapsApiModule.py

```diff
@@ -8,12 +8,15 @@
 from datetime import datetime
 import calendar
 
-
-gmaps = googlemaps.Client(config.key)  # global variable gmaps
-
 log = logging.getLogger(__name__)
 log.info("Entered module: %s" % __name__)
 
+try:
+    gmaps = googlemaps.Client(config.key)  # global variable gmaps
+except ValueError as e:
+    logging.debug(e)
+    gmaps = None
+
```

Comment:
Created a try-except block that assigns `None` to `gmaps` if Invalid Google Maps API key is provided. This is a temporary workaround for Travis CI.